.travis.yml

```diff
@@ -27,23 +31,10 @@ before_script:
     - sh -e /etc/init.d/xvfb start
     - npm i
     - node_modules/.bin/gulp build
+    - if [ $FX_VERSION = "54.0b" ] && [ $TRAVIS_REPO_SLUG = "zotero/zotero" ] && [ $TRAVIS_BRANCH = "master" ] && [ $TRAVIS_PULL_REQUEST = "false" ]; then
+        mkdir build-zip && cd build && zip -r ../build-zip/$TRAVIS_COMMIT.zip * && cd .. &&
+        dpl --provider=s3 --access-key-id=AKIAJFDVJ54MCAEXPQ5Q --bucket=zotero-download --local-dir=build-zip --upload-dir=ci/client --acl=public-read --skip_cleanup=true;
```

Comment:
Hmm. what exactly to indent? mkdir and dpl is in the same level as wget at the top. Or do you mean to move each command to separate line?